.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -90,6 +90,15 @@ update_initrd_kernel_modules()
 	( cd "${rootfs_dir}" && find . | cpio -H newc -o | gzip -9 ) > ${initrd_path}
 }
 
+# Manage symlinks on host.
+handle_symlinks() {
+	local create_links_script="/tmp/create_links.sh"
+	cp /create_links.sh "${HOST_MOUNT}/${create_links_script}"
+
+	chmod +x "${HOST_MOUNT}/${create_links_script}"
```

Comment:
Nit: You could combine these by using `install(1)` (which might be safer as you could also set perms and ownership explicitly).

Reply:
Good idea. Old habits are hard to die, `install(1)` is the right tool here. Thanks for the tip @jodh-intel !